README.md

```diff
@@ -1,40 +1,206 @@
+
+```js
+fixture `Getting Started`
+    .page('http://devexpress.github.io/testcafe/example');
+```
+
+Then create the [test](http://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.md#tests) function where place test code.
+The test will type a developer name into a text editor, click the Submit button and check a header text on the resulting page.
+
+```js
+import { expect } from 'chai';
+
+fixture `Getting Started`
+    .page('http://devexpress.github.io/testcafe/example');
+
+test('My first test', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+
+    expect((await t.select('#article-header')).innerText).to.equal('Thank you, John Smith!');
+});
+```
+
+## Running the Test
+
+You can simply run the test from a command shell, by calling a single command where you specify the [target browser](http://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.md#browser-list) and [file path](http://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.md#file-pathglob-pattern).
+
+```bash
+testcafe safari test1.js
+```
+
+TestCafe will automatically open the chosen browser and start the test execution within it.
+
+For more information on how to configure the test run, see [Command Line Interface](http://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.md).
+
+## Viewing the Test Results
+
+While the test is running, TestCafe is gathering information about the test run and outputing the report right into a command shell.
+
+![Test Report](docs/articles/images/report.png)
+
+For more information, see [Reporters](http://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/reporters.md).
+
+# Documentation
+
+- [Getting Started](http://devexpress.github.io/testcafe/documentation/getting-started/)
+- [Test API](http://devexpress.github.io/testcafe/documentation/test-api/)
+- [Using TestCafe](http://devexpress.github.io/testcafe/documentation/using-testcafe/)
+- [Extending TestCafe](http://devexpress.github.io/testcafe/documentation/extending-testcafe/)
+- [Recipes](http://devexpress.github.io/testcafe/documentation/recipes/)
+
+# Contributing
 
- - [testcafe-hammerhead](https://github.com/DevExpress/testcafe-hammerhead) - the heart.
- - [testcafe-browser-natives](https://github.com/DevExpress/testcafe-browser-natives) - the library for performing platform-dependent actions on browser windows.
- - [testcafe-legacy-api](https://github.com/DevExpress/testcafe-legacy-api) - the legacy API support.
- - [testcafe-gh-page-assets](https://github.com/DevExpress/testcafe-gh-page-assets) - the website assets.
+Please use our [issues page](https://github.com/DevExpress/testcafe/issues) to report a bug or request a feature.
```

Comment:
For general purpose questions and discussions use https://testcafe-discuss.devexpress.com/